### PR TITLE
dataio-tiff: don't fail tiff export with negative MD_POS, test tiff export for time correlator data

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

File: src/odemis/dataio/test/tiff_test.py [hunk 1]
```diff
@@ -1526,6 +1526,137 @@ def testRGB(self):
...
+        self.assertEqual(len(rdata), len(ldata))
+
+        for i, im in enumerate(rdata):
+            self.assertEqual(im.shape, shape[i])
+            self.assertEqual(im.dtype, dtype)
+            emd = metadata[i].copy()
+            rmd = im.metadata
+            img.mergeMetadata(emd)
+            img.mergeMetadata(rmd)
+            self.assertEqual(rmd[model.MD_DESCRIPTION], emd[model.MD_DESCRIPTION])
```
Comment (pieleric) [inline src/odemis/dataio/test/tiff_test.py]: Even checking MD_POS should work, as it uses the OME metadata.

File: src/odemis/dataio/test/tiff_test.py [hunk 2]
```diff
@@ -1526,6 +1526,137 @@ def testRGB(self):
...
+            self.assertEqual(rmd[model.MD_DIMS], emd[model.MD_DIMS])
+
+    def testBadTIFFMD(self):
+        """
+        Checks that can both write and read back anXXXX
```
Comment (pieleric) [inline src/odemis/dataio/test/tiff_test.py]: Please fill in the XXX, to explain which bug we are testing against.

File: src/odemis/dataio/test/tiff_test.py (hunk 1 above)
Comment (philipwinklerdelmic) [reply src/odemis/dataio/test/tiff_test.py]: done

File: src/odemis/dataio/test/tiff_test.py (hunk 2 above)
Comment (philipwinklerdelmic) [reply src/odemis/dataio/test/tiff_test.py]: done

**Pull request merged**